### ISSUE TITLE
Direct Deposit: Confirmation Screen 

### ISSUE THREAD

**Issue by bridgethapner** [department-of-veterans-affairs/va.gov-team#13649]
## User Story
As a user, I can review the direct deposit information I entered before I save it. From the confirmation screen, I can either edit information or save it. 

_This is an accessibility requirement._ 

## Tasks
- [x] High fidelity prototype of screens in iOS
- [ ] Usability test as needed, refine as needed
- [ ] (Ongoing) UI for a range of devices we are supporting, Android-specific designs as needed

**Comment by srmyers17** [comment]
@bessgreen to look into accessibility requirements for this one.

**Comment by b-rocha** [comment]
Hi @srmyers17 @bridgethapner, Bess pinged me about this so replying here! The requirement is that a user should be able to reverse, check, or confirm the financial information they’ve entered. Meaning, they should be able to perform at least one of those actions in case they’ve made a mistake. 

Since a user can edit their direct deposit info at any point in time, they can essentially “reverse” any info they’ve previously entered. WCAG also states that it’s not their intent to require a confirmation for a save command or editing records and other data. 

This would be a good thing to run by the 508 office/ an accessibility SME to confirm that this is sufficient. If they’d like to add extra safeguards by requiring the user to review/ confirm, this could be done a few ways:

-Adding a checkbox with a label like “I confirm that this information is correct” or “I have checked that this information is right” 
-Adding a confirmation dialog 
-Adding a new page/ view that shows the user a “summary” of the direct deposit info they’ve entered and requiring them to confirm (don’t love this one from a UX standpoint though).

**Comment by bridgethapner** [comment]
Thank you so much for this context, Brenda! 

I've added a checkbox to the Direct Deposit edit state saying "I confirm that this information is correct" that the user must select to Save. For MVP let's proceed with this addition for now. We can re-address if the 508 office does not think this is sufficient. They will be reviewing our work a couple of times between now and November. cc @bessgreen

**Comment by bessgreen** [comment]
Thanks @bridgethapner - since we already [estimated](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/13907) the original design w/o the confirmation checkbox, I'm creating a [separate story](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/14714) to add the checkbox.

**Comment by bridgethapner** [comment]
Sounds great, thanks @bessgreen
